mobile/src/main/java/com/truckmuncher/truckmuncher/vendor/VendorHomeActivity.java

```diff
@@ -111,4 +133,63 @@ public void run() {
             }
         });
     }
+
+    @Override
+    public Loader<Cursor> onCreateLoader(int id, Bundle args) {
+        WhereClause whereClause = new WhereClause.Builder()
+                .where(PublicContract.Truck.OWNED_BY_CURRENT_USER, EQUALS, 1)
+                .build();
+        String[] projection = TrucksOwnedByUserQuery.PROJECTION;
+        Uri uri = Contract.TRUCK_PROPERTIES_URI;
```

Comment:
Since it's a query, use `PublicContract.TRUCK_URI`. The internal ones in the `Contract` class are only there b/c we can't do a write against a SQL view